Examples/UIExplorer/NavigatorIOSColorsExample.js

```diff
@@ -66,6 +66,7 @@ var NavigatorIOSColors = React.createClass({
         tintColor="#FFFFFF"
         barTintColor="#183E63"
         titleTextColor="#FFFFFF"
+        fontFamily="Helvetica"
```

Comment:
I would be surprised if that were true, but if so then I agree fontFamily would make sense for the name.